packages/nextcloud/lib/src/client.dart

```diff
@@ -18,7 +18,7 @@ import 'package:nextcloud/src/webdav/client.dart';
 class NextcloudClient extends DynamiteClient {
   // ignore: public_member_api_docs
   NextcloudClient(
-    super.baseURL, {
+    final String baseURL, {
```

Comment:
Why not use Uri here?

Reply:
It's a pain to migrate our login stuff and as it is fragile enough I didn't change this.
Should this also be included in this PR?

Reply:
I think it would be better to do so